auditbeat/module/file_integrity/metricset_test.go

```diff
@@ -143,6 +147,8 @@ func TestActions(t *testing.T) {
 		ioutil.WriteFile(updatedFilepath, []byte("hello world"), 0600)
 	}()
 
+	time.Sleep(2 * time.Minute)
```

Comment:
Better replace the go-routine with a code block:

```
		ioutil.WriteFile(createdFilepath, []byte("hello world"), 0600)
		ioutil.WriteFile(updatedFilepath, []byte("hello world"), 0600)
```

WriteFile returns when the file is written... no need to block the test suite for an extra 2 minutes.